shared/typetracking/codeql/typetracking/internal/TypeTrackingImpl.qll

```diff
@@ -70,6 +70,10 @@ module TypeTracking<LocationSig Location, TypeTrackingInput<Location> I> {
 
   private class ContentOption = ContentOption::Option;
 
+  private predicate isLocalSourceNode(LocalSourceNode n) {
```

Comment:
This is a small helper for `standardFlowsTo`. I think it should be moved down, so it appears directly before `standardFlowsTo`. Also, check whether the RA is affected by manually inlining this - if not, then that simplifies things.